packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -947,24 +948,66 @@ abstract class ResidentRunner {
-      printStatus('To dump the rendering tree of the app (debugDumpRenderTree), press "t".');
+      printCommandHelp('w', 'Dump widget hierarchy to the console.',
+          inParenthesis: 'debugDumpApp');
+      printCommandHelp('t', 'Dump rendering tree to the console.',
+          inParenthesis: 'debugDumpRenderTree');
       if (isRunningDebug) {
-        printStatus('For layers (debugDumpLayerTree), use "L"; for accessibility (debugDumpSemantics), use "S" (for traversal order) or "U" (for inverse hit test order).');
-        printStatus('To toggle the widget inspector (WidgetsApp.showWidgetInspectorOverride), press "i".');
-        printStatus('To toggle the display of construction lines (debugPaintSizeEnabled), press "p".');
-        printStatus('To simulate different operating systems, (defaultTargetPlatform), press "o".');
-        printStatus('To toggle the elevation checker, press "z".');
+        printCommandHelp('L', 'Dump layer tree to the console.',
+            inParenthesis: 'debugDumpLayerTree');
+        printCommandHelp('S', 'Dump accessibility tree in traversal order.',
+            inParenthesis: 'debugDumpSemantics');
+        printCommandHelp(
+            'U', 'Dump accessibility tree in inverse hit test order.',
+            inParenthesis: 'debugDumpSemantics');
+        printCommandHelp('i', 'Toggle widget inspector.',
+            inParenthesis: 'WidgetsApp.showWidgetInspectorOverride');
+        printCommandHelp('p', 'Toggle the display of construction lines.',
+            inParenthesis: 'debugPaintSizeEnabled');
+        printCommandHelp('o', 'Simulate different operating systems.',
+            inParenthesis: 'defaultTargetPlatform');
+        printCommandHelp(
+            'z', 'Toggle elevation checker.');
       } else {
-        printStatus('To dump the accessibility tree (debugDumpSemantics), press "S" (for traversal order) or "U" (for inverse hit test order).');
+        printCommandHelp('S', 'Dump accessibility tree in traversal order.',
+            inParenthesis: 'debugDumpSemantics');
+        printCommandHelp(
+            'U', 'Dump accessibility tree in inverse hit test order.',
+            inParenthesis: 'debugDumpSemantics');
       }
-      printStatus('To display the performance overlay (WidgetsApp.showPerformanceOverlay), press "P".');
-      printStatus('To enable timeline events for all widget build methods, (debugProfileWidgetBuilds), press "a"');
-    }
-    if (flutterDevices.any((FlutterDevice d) => d.device.supportsScreenshot)) {
-      printStatus('To save a screenshot to flutter.png, press "s".');
+      printCommandHelp('P', 'Toggle performance overlay.',
+          inParenthesis: 'WidgetsApp.showPerformanceOverlay');
+      printCommandHelp(
+          'a', 'Toggle timeline events for all widget build methods.',
+          inParenthesis: 'debugProfileWidgetBuilds');
+    }
+  }
+
+  void printCommandHelp(String key, String description,
+      {String inParenthesis}) {
```

Comment:
Prefer:

```dart
  void printCommandHelp(String key, String description, {
￼    String inParenthesis,
  }) {
```

(again, dartfmt will do this, or something close to it, if you use trailing commas)